stable/concourse/README.md

```diff
@@ -113,6 +113,7 @@ The following table lists the configurable parameters of the Concourse chart and
 | `web.nodeSelector` | Node selector for web nodes | `{}` |
 | `web.service.type` | Concourse Web service type | `ClusterIP` |
 | `web.service.annotations` | Concourse Web Service annotations | `nil` |
+| `web.service.loadBalancerIP` | Concourse Web Service Load Balancer IP | `nil` |
```

Comment:
The IP to use when web.service.type is LoadBalancer would be more descriptive here.